geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/GeneralShapefileIngest.scala

```diff
@@ -33,10 +33,10 @@ object GeneralShapefileIngest {
   }
 
   // The goal of this method is to allow for URL-based look-ups.
-  //  This allows for us to ingest files from HDFS and S3.
+  //  This allows for us to ingest files from HDFS, S3, and Azure (WASBS and WASB).
   def getShapefileDatastore(shapefilePath: String): FileDataStore = {
     // NOTE this regex is designed to work for s3a, s3n, etc.
-    if (shapefilePath.matches("""\w{3,4}:\/\/.*$""")) {
+    if (shapefilePath.matches("""\w{3,5}:\/\/.*$""")) {
```

Comment:
any reason we don't just use a ```startsWith``` and use ```IngestCommand.RemotePrefixes``` or something along those lines? It would be more self documenting and less arbitrary to do that. Also, I feel that what if I give it ```file:///``` one day what happens?